app/scripts/lib/auth-errors.js

```diff
@@ -187,6 +187,10 @@ function (_, Errors) {
     INVALID_EMAIL: {
       errno: 1023,
       message: t('Valid email required')
+    },
+    FORCE_AUTH_EMAIL_REQUIRED: {
```

Comment:
How about: always message "Unexpected error" to the user for dev errors, then log the actual details to console and metrics.